src/main/java/io/supertokens/webserver/api/core/UsersCountAPI.java

```diff
@@ -70,7 +70,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IO
         }
 
         try {
-            long count = AuthRecipe.getUsersCount(this.getTenantIdentifier(req), super.main,
+            long count = AuthRecipe.getUsersCount(this.getTenantIdentifierStorageFromRequest(req),
                     recipeIdsEnumBuilder.build().toArray(RECIPE_ID[]::new), includeAllTenants);
```

Comment:
this will have to changed cause it will not be tenant specific if includeAllTenants is true.

Reply:
Have separated them into multiple functions and added placeholders, will finish proper impl while working on Auth recipe.